oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -114,6 +114,11 @@ organization's Core Contributors, however, we recognize that circumstances
 differ and a different commitment may be more appropriate. Before signing the
 agreement, please `Contact the Program Administrators`_ to discuss.
 
+The scope of the work or contributions considered as part of the Core 
+Contributors duties, and counted towards an organization's commitments, is
+defined at 
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3593502844/Core+Contributor+-+Work+Scope>`_
```

Comment:
The link syntax is malformed, I think you're missing the initial backtick.  It should render correctly in GitHub via "View File" from the PR once fixed, or you can check the Sphinx rendering locally.  But word of warning, I just wrote a small essay on how to do that and what it says about the current state of our developer experience: https://docs.google.com/document/d/1tM6JLczGRTnvaABhWclAPe22urzYdqG39PQk-tixIv8/edit#

Reply:
@jmbowman Thank you for the review - and sorry that it caused you such a hassle to review. :/ Even without running the whole build like you did, I could have caught it in the github render. And now I know to look for the rendered version in the tests - here https://open-edx-proposals--417.org.readthedocs.build/en/417/processes/oep-0054-core-contributors.html#core-contributor-program-administration

I'm currently blocked on not being able to update the branch from my PR, but I'll make sure to fix this once it's solved.

Reply:
@jmbowman The typo show now be fixed - you can see the current output at https://open-edx-proposals--417.org.readthedocs.build/en/417/processes/oep-0054-core-contributors.html#core-contributor-program-administration